clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -538,7 +538,7 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "npm run -S build -- --minify",
+    "vscode:prepublish": "npm run -S build -- --minify && npm run -S build:web -- --minify",
```

Comment:
```suggestion
    "vscode:prepublish": "npm run -S build -- --minify && npm run -S build:web -- --production",
```
?

Reply:
Added the flag for production but also kept the minify flag.